.gitignore

```diff
@@ -21,4 +21,6 @@ web/scripts/codemirror.js
 
 last_prod/
 
+# IDE-specific files
 .idea/
+.vscode
```

Comment:
Why not configure this [globally](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) rather than ignore commit for every project? This adds a bit of a maintenance burden for all of our OS projects.

Reply:
Reverted the change.